workspaces/bulk-import/plugins/bulk-import/src/api/BulkImportBackendClient.ts

```diff
@@ -159,7 +193,7 @@ export class BulkImportBackendClient implements BulkImportAPI {
     const { token: idToken } = await this.identityApi.getCredentials();
     const backendUrl = this.configApi.getString('backend.baseUrl');
     const jsonResponse = await fetch(
-      `${backendUrl}/api/bulk-import/import/by-repo?repo=${repo}&defaultBranch=${defaultBranch}`,
+      `${backendUrl}${this.pathProvider.getDeleteImportActionPath(repo, defaultBranch)}`,
```

Comment:
@This looks to me that we should think about multiple clients. (Same Backstage API but multiple implementations.) But we don't need to do it now.